site2/docs/io-hdfs2-sink.md

```diff
@@ -0,0 +1,53 @@
+---
+id: io-hdfs2-sink
+title: HDFS2 sink connector
+sidebar_label: HDFS2 sink connector
+---
+
+The HDFS2 sink connector pulls the messages from Pulsar topics 
+and persists the messages to HDFS files.
+
+## Configuration
+
+The configuration of the HDFS2 sink connector has the following properties.
+
+### Property
+
+| Name | Type|Required | Default | Description 
+|------|----------|----------|---------|-------------|
+| `hdfsConfigResources` | String|true|  | A file or a comma-separated list containing the Hadoop file system configuration.<br/><br/>**Example**<br/>'core-site.xml'<br/>'hdfs-site.xml' |
+| `directory` | String | true | |The HDFS directory where files read from or written to. |
+| `encoding` | String |false | |The character encoding for the files.<br/><br/>**Example**<br/>UTF-8<br/>ASCII |
+| `compression` | Compression |false | |The compression code used to compress or de-compress the files on HDFS. |
+| `kerberosUserPrincipal` |String| false| |The principal account of Kerberos user used for authentication. |
+| `keytab` | String|false|| The full pathname of the Kerberos keytab file used for authentication. |
+| `filenamePrefix` |String| false | |The prefix of the files created inside the HDFS directory.<br/><br/>**Example**<br/> The value of topicA result in files named topicA-. |
+| `fileExtension` | String| false | | The extension added to the files written to HDFS.<br/><br/>**Example**<br/>'.txt'<br/> '.seq' |
+| `separator` | char|false | |The character used to separate records in a text file. <br/><br/>If no value is provided, the contents from all records are concatenated together in one continuous byte array. |
+| `syncInterval` | long| false || The interval between calls to flush data to HDFS disk in milliseconds. |
+| `maxPendingRecords` |int| false|Integer.MAX_VALUE |  The maximum number of records that hold in memory before acking. <br/><br/>Setting this property to 1 makes every record send to disk before the record is acked.<br/><br/>Setting this property to a higher value allows buffering records before flushing them to disk. 
+
```

Comment:
@tuteng the properties from line 18 - 27 does not have default values? 
(I do not find their default values in [AbstractHdfsConfig.java](https://github.com/apache/pulsar/blob/master/pulsar-io/hdfs2/src/main/java/org/apache/pulsar/io/hdfs2/AbstractHdfsConfig.java))

Reply:
`syncInterval` default is 0  
`separator` `fileExtension` `filenamePrefix` `keytab` `kerberosUserPrincipal` `compression` these fields do not need to be added with default values.  
`hdfsConfigResources` `directory` these fields are required, there should be no need to add default values

Reply:
Got, thank you, I've incorporated.